src/ot_orchestration/operators/batch/__init__.py

```diff
@@ -1 +1,21 @@
-"""Batch operators."""
+r"""Operators for batch job.
+
+The idea behind the batch job operators is to provide a generic way to create batch jobs.
+
+The `BatchIndexOperator` is a [facade](https://refactoring.guru/design-patterns/facade) class that allows for finding the manifest generator class
+by it's label `manifest_generator_label` and then generating the batch index using discovered generator and it's specification defined in `manifest_generator_specs`.
```

Comment:
```suggestion
by its label `manifest_generator_label` and then generating the batch index using discovered generator and its specification defined in `manifest_generator_specs`.
```

I assume you meant to use the possessive form.

Reply:
yeah!